chain-signatures/node/src/protocol/consensus.rs

```diff
@@ -81,15 +81,18 @@ pub trait ConsensusProtocol {
         self,
         ctx: C,
         contract_state: ProtocolState,
+        cfg: Config,
     ) -> Result<NodeState, ConsensusError>;
 }
 
 #[async_trait]
 impl ConsensusProtocol for StartedState {
+    #[allow(unused_variables)]
     async fn advance<C: ConsensusCtx + Send + Sync>(
         self,
         ctx: C,
         contract_state: ProtocolState,
+        cfg: Config,
```

Comment:
just use `_` for the unused variable, that way we don't miss anything if we change this later